content/md/connectivity/manage-event-subscription.md

```diff
@@ -0,0 +1,83 @@
+---
+id: manage-event-subscription
+themes: connectivity, import-export-data
+title: Subscribe to **events**
+popular: true
+ee-only: false
+related: what-is-the-event-subscription, how-to-connect-my-pim, manage-your-connections
+---
+
+::: info
+Not sure about what the event subscription is? You can find all the related information [right here](what-is-the-event-subscription.html).
+::: 
+
+
+So, you're installing your first event-enabled extension and wondering how to make sure it gets everything it needs to work? 
```

Comment:
```suggestion
So, you are setting up your first event-enabled extension and you are wondering how to make sure it has everything it needs to work? 
```
Setting up instead of installing?

Reply:
```suggestion
You are about to set up your first event-enabled extension and you want to make sure it works. 
```